### DISCUSSION TITLE
PoolLab 2.0

### DISCUSSION THREAD

**Discussion by BaaderA** [dala318/python_poollab#27] [General]
Hi, please can you tell me if the integration also supports the new version of PoolLab 2.0? Thank you for the answer.

**Comment by pixbalance** [comment]
Same question here. 😊

**Comment by BaaderA** [comment]
I'll answer for myself. Yes, it works...

**Reply by cloudbr34k84** [reply]
@BaaderA How often are you testing with the poollab?
